api_docs/api_specs.json

```diff
@@ -469,6 +469,56 @@
         "comment": "Response acknowledging the successful deletion of the config",
         "schema": "default_message.json"
       }
+    },
+    "bridge_update_check": {
+      "title": "Bridge update check",
+      "description": "Checks whether the bridge has an update available. Responds with bool value for available update and branch information",
+      "broadcast_allowed": true,
+      "http_method": null,
```

Comment:
Since the functionality should be accessible via web interface this needs to be set (GET in this case I guess?)

Reply:
method set to "GET"